ShelterModule/appsettings.json

```diff
@@ -6,7 +6,5 @@
     }
   },
   "AllowedHosts": "*",
-  "ConnectionStrings": {
-    "PetShareDatabase": "Server=localhost;Integrated Security=true;DataBase=PetShare;TrustServerCertificate=true"
-  }
-}
+  "KeyVaultURL": "https://petsharekeyvault.vault.azure.net/"
+}
```

Comment:
Let's add a newline here at the end, cuz github says it's wrong and marks it with an evil minus sign